Procfile

```diff
@@ -1,10 +1,10 @@
-publisher: cd publisher && LC_ALL=en_GB.UTF-8 exec govuk_setenv publisher bundle exec rackup -p 3079
```

Comment:
Can we leave this in or will it fail if `/etc/govuk/publisher/env.d` is missing on the box?